File: WebApp/Models/Cenovnik.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace JGSPNSWebApp.Models
{
    public class Cenovnik
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public DateTime VaziOd { get; set; }

        public DateTime VaziDo { get; set; }

        public bool  Aktivan { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public List<Stavka> Stavke { get; set;}


    }
}

File: AngularClientApp/src/app/Models/Cenovnik.ts
import { Stavka } from './Stavka';

export class Cenovnik{
    //public Id:number;
    public VaziOd: Date;
    public VaziDo: Date;
    public Aktivan: boolean;
}

export class CenovnikPrikaz{
    public Id:number;
    public VaziOd: Date;
    public VaziDo: Date;
    public Stavke: Stavka[];
    public Aktivan: boolean;
    public Version: number;
  
}

export class CenovnikUpdate{
    public Id:number;
    public VaziOd: Date;
    public VaziDo: Date;
    public Aktivan: boolean;
    public Version: number;
  
}
